docs: update EFB performance calculators (#1080)
flybywiresim/docs: docs/aircraft/common/flypados3/performance.md

**Runway**

> - Select the runway identifier of the runway being used for takeoff.

Reviewer: I feel like this language overall could be a bit better but otherwise good

Reviewer: Something like "Select the desired runway identifier for the takeoff"?

the airport 10-9 chart, but requires a little calculation from the data present on Jeppesen charts, as shown in the examples in [Advanced Takeoff Calculator Guide](../../../pilots-corner/a32nx/a32nx-advanced-guides/flight-planning/takeoff-perf-calc.md). A negative slope is downhill, while a positive slope is uphill.

**Entry Angle** 

- This is the angle of the taxiway to the runway. An input is needed to allow the calculator to reduce the TORA distance by the distance used to align the airplane on 
the runway for takeoff (since the runway behind the airplane should not be used to calculate takeoff performance). The entry angle can be determined from the airport 10-9 (or 10-9A) chart as shown in the examples in [Advanced Takeoff Calculator Guide](../../../pilots-corner/a32nx/a32nx-advanced-guides/flight-planning/takeoff-perf-calc.md).

**Runway Condition** 

- This allows input of runway conditions, including dry, wet, and numerous types of runway contamination. It defaults to a dry runway. Although the calculations are provided 

> for other runway conditions, be aware that MSFS 2020 currently does not simulate these surface conditions properly. The runway condition can be reported in the ATIS or in a SNOWTAM.

Reviewer: ```suggestion
for other runway conditions, be aware that MSFS 2020 currently does not simulate these surface conditions properly. The runway condition can be found on an ATIS report or in a SNOWTAM.
```

### Usage

Enter the required data into the calculator fields. The initial state of the calculator will have all input fields blank except for runway condition (Dry), flap setting (Full), overweight procedure (No), reverse thrust (No) and autoland (No). 
It is important to note that all fields must be filled to activate the "Calculate" button. The "Clear" button will return input fields to the initial state. 

Weather data can be automatically filled by entering an airport ICAO and pressing "Get METAR". How to get the information for all the fields is described in the following section.

### Information Required

**Airport**

- Enter the ICAO code for the airport.

**Runway**

- Select the runway identifier of the runway being used for landing.

**Runway Heading** 

- This is the direction of the runway referenced to magnetic North. It will be filled automatically after selecting the runway.

> **Runway LDA**

Reviewer: Can we add a highlight of what LDA means using the underline abbreviation text system? Would be useful IMO

Author: Already included in the changes

from the MSFS 2020 runway length data and may not always be accurate. We recommend checking the LDA distance with the airport chart as shown in the example below.

**Runway Elevation** 

- This is the elevation of the runway at the threshold. It will be automatically filled after selecting the runway. It can also be determined or checked from your final approach charts.

**Runway Slope** 

- This is the gradient of the runway in the direction of takeoff. It will be automatically entered after selecting the runway. It can also be determined or checked from 
the airport chart, but requires a little calculation from the data present on Jeppesen charts (see example in [Landing Calculator Additional Info](./#example-chart)) A negative slope is downhill, while a positive slope is uphill.
- Formula: (((Runway elevation on touchdown side) - (Runway elevation on far side)) / Runway length) * 100.
- E.g., EDDM 26R: 1449 ft - 1467 ft = - 18 ft.
- Then - 18 ft / 13 123 ft = - 0.001 37.
- Then - 0.00137 * 100 = - 0.137 % slope.

**Runway Condition** 

- This allows input of runway condition codes, from dry (6) to poor (1). It defaults to a dry runway. Although the calculations are provided 

> for other runway conditions, be aware that MSFS 2020 currently does not simulate these surface conditions properly. The runway condition can be reported in the ATIS or in a SNOWTAM. For detailed information about RCC's see the [RWY Conditions](../../../pilots-corner/airliner/airliner-flying-guide/runway-conditions.md)

Reviewer: ```suggestion
for other runway conditions, be aware that MSFS 2020 currently does not simulate these surface conditions properly. The runway condition can be found on an ATIS report or in a SNOWTAM. For detailed information about RCC's see the [RWY Conditions](../../../pilots-corner/airliner/airliner-flying-guide/runway-conditions.md)
```
1. same as earlier 

2. same as above, should we have the RCC indicate what it means with an underscore (PS: I think this is done already on the bottom of the page right?)

**Temperature** 

- This is simply the outside air temperature. It can be automatically entered from either the OFP or the METAR as described above. It can also be entered manually based on 
either the active ATIS or METAR.

**QNH** 

- This is the atmospheric pressure at the airport, adjusted for elevation above mean sea level. It can be automatically filled from either the OFP or the METAR as described above. It can also be entered manually 
based on either the active ATIS or METAR.

**Approach Speed: MCDU PERF APPR**
- This is part of the MCDU PERF APPR page and can be taken from there (V~APP~).

![MCDU PERF APPR page](../assets/flypados3/performance-landing-mcdu-perf.png "MCDU PERF APPR page"){loading=lazy}

**Weight**
- This information can be obtained from the ECAM SD bottom-right corner (GW 59 600 kg in the example below).

![ECAM SD](../assets/flypados3/performance-landing-ecam-sd.png "ECAM SD"){loading=lazy}

**Flaps**
- Planned landing flap setting (FULL or CONF 3).

**Overweight Procedure**
- Will the landing weight be greater than the maximum landing weight (Yes or No).

> - Reminder: maximum landing weight is 67 400 kg.

Reviewer: Since this is a common page for both planes, I recommend we show this weight is for the A32NX and also list the A380X too

**Reverse Thrust**
- Will reverse thrust be used (Yes or No)?
- Use of reverse thrust on a wet or contaminated runway would be considered necessary for safety reasons.
- May be subject to airport requirements. If there are airport requirements regarding the use of reverse thrust, these can be found on the airport chart (see example below).

![Airport Chart Information](../assets/flypados3/performance-landing-charts-reverser.png "Airport Chart Information"){loading=lazy}

**Autoland**

> - If an Autoland is planned, then set this to Yes.

Reviewer: to follow the earlier format: 
```suggestion
- Is Autoland planned to be used (Yes or No)?
```
